### PR TITLE
Fix tf model persistor and tf model

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
Fix tf model persistor and tf model

### Description

- add missing argument
- allow passing persistor object

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/app_opt/tf/job_config/model.py [hunk 1]
```diff
@@ -12,22 +12,30 @@
...
         If model is a tf.keras.Model, add a TFModelPersistor with the model.
 
         Args:
             model (any): model
+            persistor (Optional[TFModelPersistor]): A TFModelPersistor,
+            if provided will ignore argument `model`, defaults to None.
         """
         self.model = model
 
+        validate_object_for_job("persistor", persistor, TFModelPersistor)
```
Comment (SYangster) [inline nvflare/app_opt/tf/job_config/model.py:36]: we should check `if persistor is not None:` before validating

Comment (YuanTingHsieh) [reply nvflare/app_opt/tf/job_config/model.py:36]: good catch, let me fix it.

**Review by SYangster — APPROVED** [review]
Thanks LGTM

**Pull request merged**